storage/tests/collision.rs

```diff
@@ -0,0 +1,32 @@
+mod common;
+
+use {
+    eth_trie::DB,
+    hex_literal::hex,
+    moved::primitives::B256,
+    moved_storage::{RocksEthTrieDb, ROOT_KEY},
+};
+
+#[test]
+fn test_column_families_do_not_collide() {
+    let rocks = common::create_db();
+    let db = RocksEthTrieDb::new(&rocks);
+
+    let expected_root = B256::new(hex!(
+        "50596cee391a497683672d9396379f56cd8e96476b844557933f48039c483a81"
```

Comment:
Is this the state root after adding `0xfeef`? If so can you leave a short comment above describing what this root value is.

Reply:
I feel it would be a pain to maintain this, as with every change of the state root this test will need to be updated too. Maybe it warrants a separate issue for making it more ergonomic?

Reply:
It's random bytes, trie logic does not affect this test.